### DISCUSSION TITLE
List of all attributes generated by httpx.get()

### DISCUSSION THREAD

**Discussion by Tosh0kan** [encode/httpx#2584] [Q&A] [answered]
I searched the documentation but didn't find anything related to it. Is there a list of all the attributes that the object generated by `httpx.get()` has. For example, when I run the code in debug and pause, I can see it has attributes like `is_client_error`, or `num_bytes_downloaded`, or `status_code`, so I was wondering if somewhere in the documentation, that has all these characteristics listed.

**Comment by lovelydinosaur** [comment, marked as answer]
Hi @Tosh0kan, the API listing of the `Response` instance is documented here... https://www.python-httpx.org/api/#response

But it looks like we might be missing some properties?

**Reply by Tosh0kan** [reply]
No, you don't. Sorry, I just wasn't being able to find this info in the docs. Thank you for pointing it out and sorry for the trivial question!
